File: backend/albergue1/models/tipo_mascota.py
from utils.db import db

class Tipo_mascota(db.Model):
    __tablename__ = 'tipo_mascota'

    # Variables
    id_tipo_mascota = db.Column(
        db.Integer, 
        primary_key=True,
        autoincrement=True,
        nullable=False
    )

    nombre = db.Column(
        db.String(20),
        nullable=False,
        unique=True
    )

    # Constructor
    def __init__(self,
                nombre):
        self.nombre=nombre

File: frontend/src/app/models/tipo-mascota.ts
export class TipoMascota {
  id_tipo_mascota : number;
  nombre: string;

  constructor(
    id_tipo_mascota: number,
    nombre: string
  ) {
    this.id_tipo_mascota = id_tipo_mascota;
    this.nombre = nombre;
  }
}
